bin/svg2png-cli.js

```diff
@@ -1,13 +1,19 @@
 #!/usr/bin/env node
+
+/* eslint-disable no-process-exit */
+
 "use strict";
-const fs = require("fs");
+
+const { EOL } = require("os");
```

Comment:
Please don't use OS EOLs. \n everywhere is much preferable.

Reply:
No problem. I was trying to be cross-platform friendly, but I guess it's a bit out-of-scope anyway. I'll change back.